src/chef-server-ctl/plugins/version.rb

```diff
@@ -24,6 +24,9 @@
     # isn't appropriate in all cases.  One option would be to ask the
     # LB for our version, but then the version command won't work when
     # we are offline.
+
+    puts ::ChefServerCtl::Config::DOC_PATENT_MSG
```

Comment:
I think let's drop it from here in case people are parsing the version command output.  I think that help and reconfigure line up nicely with what chef-server has done.